### ISSUE TITLE
BlackList Cars on LegacyFuel

### ISSUE THREAD

**Issue by tomasspvarela** [InZidiuZ/LegacyFuel#74]
Hi!

I don’t know why, but I can’t Blacklist cars on Legacyfuel!
Here is the code:

– Blacklist certain vehicles.
Config.Blacklist = {
“Dilettante”,
“trans_teslams”,
“Raiden”
}

Can you help me?

Ty! :)

**Comment by MewizardVT** [comment]
i don't believe that feature was implemented before the developers went awol; i might try to pick this up and keep making it

**Comment by tomasspvarela** [comment]
Ok, ty for helping!
